turbo.json

```diff
@@ -9,7 +9,8 @@
     "FONTS_REPO_ACCESS_TOKEN",
     "DATABASE_URL",
     "NODE_ENV",
-    "VITE_DATABASE_URL"
+    "VITE_DATABASE_URL",
+    "NO_STAKE_BOOST_AMOUNT"
```

Comment:
shouldnt this be in the project/package specific turbo file, and not the root one?

Reply:
Yea, i tried that in the past and had problems getting it to work. Do we have other examples of that? Been meaning to try that again and clean this all up.

Reply:
hm i wonder why? but we have [this file](https://github.com/recallnet/js-recall/blob/main/apps/api/turbo.json) in the `apps/api` dir, and it works for all of the backend related env vars.

Reply:
Went ahead and moved the one introduced in this PR